src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -593,6 +595,38 @@ export class InstalledExtensionsView extends ExtensionsListView {
 	}
 }
 
+export class EnabledExtensionsView extends ExtensionsListView {
+
+	public static isEnabledExtensionsQuery(query: string): boolean {
+		return ExtensionsListView.isEnabledExtensionsQuery(query);
+	}
+
+	async show(query: string): TPromise<IPagedModel<IExtension>> {
```

Comment:
Why not go with a simple

```
async show(query: string): TPromise<IPagedModel<IExtension>> {
		return super.show('@enabled');
	}
```

?

Same for the the `show` method in `DisabledExtensionsView `

Reply:
done! you're definitely right, I've inlined the code and removed these methods, and refined the query string checking logic a bit more, cheers!